ui/src/components/CreateRecipe.vue

```diff
@@ -177,6 +189,34 @@ const cancelEdit = () => {
                                       class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-green-500 focus:border-green-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-green-500 dark:focus:border-green-500"
                                       placeholder="Brief description of your recipe" required></textarea>
                         </div>
+
+                        <div class="mb-4">
+                            <div class="flex flex-col sm:flex-row sm:space-x-6">
+                                <!-- Preparation Time -->
+                                <div class="sm:flex-1">
+                                    <label for="preparationTime"
+                                        class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Preparation time</label>
+                                    <div class="flex items-center space-x-2">
+                                        <input v-model="preparationTime" type="number" id="preparationTime" min="0" max="3600"
+                                            class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-green-500 focus:border-green-500 p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-green-500 dark:focus:border-green-500 w-16"
+                                            placeholder="60">
+                                        <span class="text-sm text-gray-700 dark:text-gray-300">mins</span>
```

Comment:
What do you think about a dropdown with predefined values instead? 

10 min, 20 min, 30 min, 45 min, 1 hour, 1,5 hour, 2+ hours seems super reasonable to me and also makes it easier to just select one that is "somewhat close". Whether a meal takes 15 or 20 minutes doesn't really matter anyway.